My portfolio project pull request

Kashif-Saleem-Ghuman opened the pull request:
## Portfolio template replica: milestone 1

In this milestone:

-Added index.html file for displaying the elements o the portfolio on the browser
-Added style.css file for styling the elements of the page. 
-Added an image folder that contains all the images used in the HTML file.

Kashif-Saleem-Ghuman committed:
Add html file

Kashif-Saleem-Ghuman committed:
Add css file

Kashif-Saleem-Ghuman committed:
Add image folder

Abenezer-Tilahun commented on index.html:
```diff
@@ -0,0 +1,34 @@
+
+</head>
+<body>
+  <nav class="container">
+    <img id="logo" src="img/image_geometry_1.png" alt="">
+    <div id="bars">
+      <a class="active" href="#"><i class="fa fa-bars" style="color:black;"></i></a>
+    </div>
+  </nav>
+  <header class="heading">
+    <div id="name">
+      Kashif Saleem
+    </div>
+  </header>
```
- [ ]  For Making your HTML markup is well structured and follows a clear hierarchy and semantic correct I recommend you to use `header` tag for Toolbar section with in `nav` tag.

Kashif-Saleem-Ghuman replied:
Do you mean that I should replace the **nav** tag with a **header** tag?

Abenezer-Tilahun replied:
Hello @Kashif-Saleem-Ghuman `nav` tag should be inside the `header` tag.

Abenezer-Tilahun commented on index.html:
```diff
@@ -0,0 +1,34 @@
+      Kashif Saleem
+    </div>
+  </header>
+  <intro class="intro">
+    <p id="para">
+      Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+      If you like what you see and have a project you need coded, don’t hesitate and contact me.
+    </p>
+  </intro>
+  <div class="button">
+    <a id="btn" href="">Start collaboraton</a>
+  </div>
```
- [ ] For Making your HTML markup is well structured and follows a clear hierarchy and semantic correct I recommend you to use `section` tag for headline section.
- [ ] I recommend you to use `h1` tag for main title("Hello! I am a software developer!") and `p` tag for paragraph("I can help you build a product, feature or website...")

Kashif-Saleem-Ghuman replied:
**Please see if this is correct now**
![Capture](https://user-images.githubusercontent.com/111677605/198226484-e6092cdb-35fc-43e5-b02d-5b07fd556411.JPG)

Kashif-Saleem-Ghuman replied:
**Please see if this is correct now**
![changed](https://user-images.githubusercontent.com/111677605/198227182-63d83f98-da0a-447c-87eb-4c2277aeaa8f.JPG)

Abenezer-Tilahun replied:
Hello @Kashif-Saleem-Ghuman  good for using `section ` tag but still i recommend you to include `button`  inside the `section` tag and the design great.

Abenezer-Tilahun reviewed (changes requested):
Hi @Kashif-Saleem-Ghuman   , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Good README 
✔️ Good commit messages

![almost](https://user-images.githubusercontent.com/65860678/170491251-b1add8ba-b18f-4648-a893-20e510d957fd.gif)

## Required Changes ♻️



_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Kashif-Saleem-Ghuman commented:
I have replied to the comments with the changes made that you requested. Kindly check if those are correct. I shall then submit it for review.

Thank You.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

From: ***@***.***>
Sent: Thursday, October 27, 2022, 12:34 PM
To: ***@***.***>
Cc: ***@***.***>; ***@***.***>
Subject: Re: [Kashif-Saleem-Ghuman/My-Portfolio-] My portfolio project pull request (PR #1)


@Abenezer-Tilahun requested changes on this pull request.

Hi @Kashif-Saleem-Ghuman<https://github.com/Kashif-Saleem-Ghuman> ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Good README
✔️ Good commit messages

[almost]<https://user-images.githubusercontent.com/65860678/170491251-b1add8ba-b18f-4648-a893-20e510d957fd.gif>

Required Changes ♻️

Check the comments below the review.

Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.


As described in the Code reviews limits policy<https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form<https://airtable.com/shrQAqnBwek5a0O0s>.


In index.html<https://github.com/Kashif-Saleem-Ghuman/My-Portfolio-/pull/1#discussion_r1006499130>:

> +  <nav class="container">



+    <img id="logo" src="img/image_geometry_1.png" alt="">



+    <div id="bars">



+      <a class="active" href="#"><i class="fa fa-bars" style="color:black;"></i></a>



+    </div>



+  </nav>



+  <header class="heading">



+    <div id="name">



+      Kashif Saleem



+    </div>



+  </header>



  *   [ ] For Making your HTML markup is well structured and follows a clear hierarchy and semantic correct I recommend you to use header tag for Toolbar section with in nav tag.


In index.html<https://github.com/Kashif-Saleem-Ghuman/My-Portfolio-/pull/1#discussion_r1006502390>:

> +  <intro class="intro">



+    <p id="para">



+      Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.



+      If you like what you see and have a project you need coded, don’t hesitate and contact me.



+    </p>



+  </intro>



+  <div class="button">



+    <a id="btn" href="">Start collaboraton</a>



+  </div>



  *   [ ] For Making your HTML markup is well structured and follows a clear hierarchy and semantic correct I recommend you to use section tag for headline section.
  *   [ ] I recommend you to use h1 tag for main title("Hello! I am a software developer!") and p tag for paragraph("I can help you build a product, feature or website...")

—
Reply to this email directly, view it on GitHub<https://github.com/Kashif-Saleem-Ghuman/My-Portfolio-/pull/1#pullrequestreview-1157737531>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A2UBBJIIB63VTTTET6J7NODWFIWATANCNFSM6AAAAAARPWOIHU>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

Kashif-Saleem-Ghuman committed:
Make changes to index.html and styl.css

Kashif-Saleem-Ghuman commented:
Ok, I have done that and submitted it for review.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

From: ***@***.***>
Sent: Thursday, October 27, 2022, 3:08 PM
To: ***@***.***>
Cc: ***@***.***>; ***@***.***>
Subject: Re: [Kashif-Saleem-Ghuman/My-Portfolio-] My portfolio project pull request (PR #1)


@Abenezer-Tilahun commented on this pull request.


In index.html<https://github.com/Kashif-Saleem-Ghuman/My-Portfolio-/pull/1#discussion_r1006674932>:

> +  <nav class="container">

+    <img id="logo" src="img/image_geometry_1.png" alt="">

+    <div id="bars">

+      <a class="active" href="#"><i class="fa fa-bars" style="color:black;"></i></a>

+    </div>

+  </nav>

+  <header class="heading">

+    <div id="name">

+      Kashif Saleem

+    </div>

+  </header>

Hello @Kashif-Saleem-Ghuman<https://github.com/Kashif-Saleem-Ghuman> nav tag should be inside the header tag.

—
Reply to this email directly, view it on GitHub<https://github.com/Kashif-Saleem-Ghuman/My-Portfolio-/pull/1#discussion_r1006674932>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A2UBBJOQDPUHZ36IHDQA5N3WFJICTANCNFSM6AAAAAARPWOIHU>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

Kashif-Saleem-Ghuman commented:
Oh but since you didn’t mention the button earlier, hence I did not include it and I have already submitted it for review.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

From: ***@***.***>
Sent: Thursday, October 27, 2022 3:10 PM
To: ***@***.***>
Cc: ***@***.***>; ***@***.***>
Subject: Re: [Kashif-Saleem-Ghuman/My-Portfolio-] My portfolio project pull request (PR #1)


@Abenezer-Tilahun commented on this pull request.


In index.html<https://github.com/Kashif-Saleem-Ghuman/My-Portfolio-/pull/1#discussion_r1006676362>:

> +  <intro class="intro">



+    <p id="para">



+      Hello! I am a software developer! I can help you build a product, feature, or website. Taka a look of my works.



+      If you like what you see and have a project you need coded, don’t hesitate and contact me.



+    </p>



+  </intro>



+  <div class="button">



+    <a id="btn" href="">Start collaboraton</a>



+  </div>



Hello @Kashif-Saleem-Ghuman<https://github.com/Kashif-Saleem-Ghuman> good for using section tag but still i recommend you to include button inside the section tag

—
Reply to this email directly, view it on GitHub<https://github.com/Kashif-Saleem-Ghuman/My-Portfolio-/pull/1#discussion_r1006676362>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A2UBBJOK7FXK5TLD5PEXND3WFJIIZANCNFSM6AAAAAARPWOIHU>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

leonardodiasb commented on index.html:
```diff
@@ -0,0 +1,39 @@
+      </div>
+    </nav>
+  </header>
+  <header class="heading">
+    <div id="name">
+      Kashif Saleem
+    </div>
+  </header>
+  <section class="intro">
+    <h1 id="intro-heading">
+      Hello! I am a software developer!
+    </h1>
+    <p id="para">
+      I can help you build a product, feature or website. Taka a look of my works.
+      If you like what you see and have a project you need coded, don’t hesitate and contact me.
+    </p>
+  </section>
```
- [ ] The header section is wrapping the nav menu in the top and the title of the page("Kashif Saleem") should be inside the first section of the page. Please move the title to the section.
 - [ ] The previous code reviewer asked you to use the `h1` tag, but in a not correct place. The "Hello! I am a software developer!" is part of the paragraph and it was correct when you first submitted the PR. The main title (e.g., "Kashif Saleem") should be an `h1` tag. If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. Kindly move your name inside the `h1` and move the "Hello..." to inside the `p` tag.

Kashif-Saleem-Ghuman replied:
**Is this correct now?**
![image](https://user-images.githubusercontent.com/111677605/198273874-fca8eedb-56d9-4b52-a006-ac8ecf5d0ffe.png)
 **For the first suggestion are you implying that I change the tag of (KASHIF SALEEM) from header to section?**

leonardodiasb commented on index.html:
```diff
@@ -0,0 +1,39 @@
+<body>
+  <header>
+    <nav class="container">
+      <img id="logo" src="img/image_geometry_1.png" alt="">
```
- [OPTIONAL] This is a background image (decorations are always background images, not `img` tags). If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.background image (decorations are always background images, not img tags).

If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

Kashif-Saleem-Ghuman replied:
**is this correct now?**
![image](https://user-images.githubusercontent.com/111677605/198275100-2ea49984-bb6a-4f07-9b73-bd1accb93408.png)

leonardodiasb reviewed (changes requested):
# Required Changes ♻️ ⚠️ 🚦 

Hi @Kashif-Saleem-Ghuman, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

![mario_ladders](https://user-images.githubusercontent.com/20563252/121369265-4e418880-c901-11eb-848c-1eff06d5a4cf.gif)

## Good points 👍 

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Good README
✔️ Good commit messages

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Kashif-Saleem-Ghuman commented:
So the code reviewer misguided me ?, and because of that, I wasted a review. Will I get an extra review now without using up my extra reviews?

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

________________________________
From: Leonardo Dias Bussular ***@***.***>
Sent: Thursday, October 27, 2022 3:18:47 PM
To: Kashif-Saleem-Ghuman/My-Portfolio- ***@***.***>
Cc: Kashif-Saleem-Ghuman ***@***.***>; Mention ***@***.***>
Subject: Re: [Kashif-Saleem-Ghuman/My-Portfolio-] My portfolio project pull request (PR #1)


@leonardodiasb requested changes on this pull request.

Required Changes ♻️ ⚠️ 🚦

Hi @Kashif-Saleem-Ghuman<https://github.com/Kashif-Saleem-Ghuman>,

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

[mario_ladders]<https://user-images.githubusercontent.com/20563252/121369265-4e418880-c901-11eb-848c-1eff06d5a4cf.gif>

Good points 👍

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Good README
✔️ Good commit messages

Aspects to improve ♻️

  *    Check the comments under the review.

Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

________________________________

As described in the Code reviews limits policy<https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using this form<https://airtable.com/shrQAqnBwek5a0O0s>.

________________________________

In index.html<https://github.com/Kashif-Saleem-Ghuman/My-Portfolio-/pull/1#discussion_r1006682483>:

> +  <header class="heading">

+    <div id="name">

+      Kashif Saleem

+    </div>

+  </header>

+  <section class="intro">

+    <h1 id="intro-heading">

+      Hello! I am a software developer!

+    </h1>

+    <p id="para">

+      I can help you build a product, feature or website. Taka a look of my works.

+      If you like what you see and have a project you need coded, don’t hesitate and contact me.

+    </p>

+  </section>



  *    The header section is wrapping the nav menu in the top and the title of the page("Kashif Saleem") should be inside the first section of the page. Please move the title to the section.
  *    The previous code reviewer asked you to use the h1 tag, but in a not correct place. The "Hello! I am a software developer!" is part of the paragraph and it was correct when you first submitted the PR. The main title (e.g., "Kashif Saleem") should be an h1 tag. If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. Check this link<https://www.reliablesoft.net/h1-tag/> for more info. Kindly move your name inside the h1 and move the "Hello..." to inside the p tag.

________________________________

In index.html<https://github.com/Kashif-Saleem-Ghuman/My-Portfolio-/pull/1#discussion_r1006683522>:

> @@ -0,0 +1,39 @@

+<!DOCTYPE html>

+<html lang="en">

+<head>

+    <meta charset="UTF-8">

+    <meta http-equiv="X-UA-Compatible" content="IE=edge">

+    <meta name="viewport" content="width=device-width, initial-scale=1.0">

+    <title>Document</title>

+    <link rel="stylesheet" href="style.css">

+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">

+

+</head>

+<body>

+  <header>

+    <nav class="container">

+      <img id="logo" src="img/image_geometry_1.png" alt="">



  *   [OPTIONAL] This is a background image (decorations are always background images, not img tags). If an image is not part of the content (aesthetic reasons) use CSS background-image. Check this link<https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image> for more info.background image (decorations are always background images, not img tags).

If an image is not part of the content (aesthetic reasons) use CSS background-image. Check this link<https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image> for more info.

—
Reply to this email directly, view it on GitHub<https://github.com/Kashif-Saleem-Ghuman/My-Portfolio-/pull/1#pullrequestreview-1158000778>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A2UBBJNZPLJIWGPLXJQGLNTWFJJIPANCNFSM6AAAAAARPWOIHU>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

Lameck1 reviewed (commented):
Hello, @Kashif-Saleem-Ghuman 👋,

_This is an additional review as a result of the second opinion request._

The concerns of @leonardodiasb are genuine. He has just tried to give further direction on what @Abenezer-Tilahun really intended to point out. Please consider addressing the concerns @leonardodiasb has raised and ask for another code review if you have any review limit left. If you don't have any review limit left, work on the concerns then leave a message. I will take over from there.

Keep rocking 🚀!

Kashif-Saleem-Ghuman commented:
Ok I have one review left so I will make changes and submit.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

From: Lameck ***@***.***>
Sent: Thursday, October 27, 2022 4:21 PM
To: ***@***.***>
Cc: ***@***.***>; ***@***.***>
Subject: Re: [Kashif-Saleem-Ghuman/My-Portfolio-] My portfolio project pull request (PR #1)


@Lameck1 commented on this pull request.

Hello, @Kashif-Saleem-Ghuman<https://github.com/Kashif-Saleem-Ghuman> 👋,

This is an additional review as a result of the second opinion request.

The concerns of @leonardodiasb<https://github.com/leonardodiasb> are genuine. He has just tried to give further direction on what @Abenezer-Tilahun<https://github.com/Abenezer-Tilahun> really intended to point out. Please consider addressing the concerns @leonardodiasb<https://github.com/leonardodiasb> has raised and ask for another code review if you have any review limit left. If you don't have any review limit left, work on the concerns then leave a message. I will take over from there.

Keep rocking 🚀! // can be customized

—
Reply to this email directly, view it on GitHub<https://github.com/Kashif-Saleem-Ghuman/My-Portfolio-/pull/1#pullrequestreview-1158084484>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A2UBBJLGOQL6BVO66ZLYTMLWFJQR5ANCNFSM6AAAAAARPWOIHU>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

Kashif-Saleem-Ghuman commented:
*   [ ] The header section is wrapping the nav menu in the top and the title of the page("Kashif Saleem") should be inside the first section of the page. Please move the title to the section.
I am unable to understand what exactly are you asking me to do here. Can u please explain to me?

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

From: Leonardo Dias ***@***.***>
Sent: Thursday, October 27, 2022 3:18 PM
To: ***@***.***>
Cc: ***@***.***>; ***@***.***>
Subject: Re: [Kashif-Saleem-Ghuman/My-Portfolio-] My portfolio project pull request (PR #1)


@leonardodiasb requested changes on this pull request.

Required Changes ♻️ ⚠️ 🚦

Hi @Kashif-Saleem-Ghuman<https://github.com/Kashif-Saleem-Ghuman>,

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

[mario_ladders]<https://user-images.githubusercontent.com/20563252/121369265-4e418880-c901-11eb-848c-1eff06d5a4cf.gif>

Good points 👍

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Good README
✔️ Good commit messages

Aspects to improve ♻️

  *   [X] Check the comments under the review.

Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.


As described in the Code reviews limits policy<https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using this form<https://airtable.com/shrQAqnBwek5a0O0s>.


In index.html<https://github.com/Kashif-Saleem-Ghuman/My-Portfolio-/pull/1#discussion_r1006682483>:

> +  <header class="heading">



+    <div id="name">



+      Kashif Saleem



+    </div>



+  </header>



+  <section class="intro">



+    <h1 id="intro-heading">



+      Hello! I am a software developer!



+    </h1>



+    <p id="para">



+      I can help you build a product, feature or website. Taka a look of my works.



+      If you like what you see and have a project you need coded, don’t hesitate and contact me.



+    </p>



+  </section>



  *   [ ] The header section is wrapping the nav menu in the top and the title of the page("Kashif Saleem") should be inside the first section of the page. Please move the title to the section.
  *   [ ] The previous code reviewer asked you to use the h1 tag, but in a not correct place. The "Hello! I am a software developer!" is part of the paragraph and it was correct when you first submitted the PR. The main title (e.g., "Kashif Saleem") should be an h1 tag. If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. Check this link<https://www.reliablesoft.net/h1-tag/> for more info. Kindly move your name inside the h1 and move the "Hello..." to inside the p tag.


In index.html<https://github.com/Kashif-Saleem-Ghuman/My-Portfolio-/pull/1#discussion_r1006683522>:

> @@ -0,0 +1,39 @@



+<!DOCTYPE html>



+<html lang="en">



+<head>



+    <meta charset="UTF-8">



+    <meta http-equiv="X-UA-Compatible" content="IE=edge">



+    <meta name="viewport" content="width=device-width, initial-scale=1.0">



+    <title>Document</title>



+    <link rel="stylesheet" href="style.css">



+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">



+



+</head>



+<body>



+  <header>



+    <nav class="container">



+      <img id="logo" src="img/image_geometry_1.png" alt="">



  *   [OPTIONAL] This is a background image (decorations are always background images, not img tags). If an image is not part of the content (aesthetic reasons) use CSS background-image. Check this link<https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image> for more info.background image (decorations are always background images, not img tags).

If an image is not part of the content (aesthetic reasons) use CSS background-image. Check this link<https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image> for more info.

—
Reply to this email directly, view it on GitHub<https://github.com/Kashif-Saleem-Ghuman/My-Portfolio-/pull/1#pullrequestreview-1158000778>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A2UBBJNZPLJIWGPLXJQGLNTWFJJIPANCNFSM6AAAAAARPWOIHU>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

Kashif-Saleem-Ghuman committed:
Change index.html and style.css as requested by leonardo

Kashif-Saleem-Ghuman commented:
***@***.***
I have made the changes. Kindly check if it is ok now?

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

________________________________
From: Lameck Otieno ***@***.***>
Sent: Thursday, October 27, 2022 4:21:02 PM
To: Kashif-Saleem-Ghuman/My-Portfolio- ***@***.***>
Cc: Kashif-Saleem-Ghuman ***@***.***>; Mention ***@***.***>
Subject: Re: [Kashif-Saleem-Ghuman/My-Portfolio-] My portfolio project pull request (PR #1)


@Lameck1 commented on this pull request.

Hello, @Kashif-Saleem-Ghuman<https://github.com/Kashif-Saleem-Ghuman> 👋,

This is an additional review as a result of the second opinion request.

The concerns of @leonardodiasb<https://github.com/leonardodiasb> are genuine. He has just tried to give further direction on what @Abenezer-Tilahun<https://github.com/Abenezer-Tilahun> really intended to point out. Please consider addressing the concerns @leonardodiasb<https://github.com/leonardodiasb> has raised and ask for another code review if you have any review limit left. If you don't have any review limit left, work on the concerns then leave a message. I will take over from there.

Keep rocking 🚀! // can be customized

—
Reply to this email directly, view it on GitHub<https://github.com/Kashif-Saleem-Ghuman/My-Portfolio-/pull/1#pullrequestreview-1158084484>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A2UBBJLGOQL6BVO66ZLYTMLWFJQR5ANCNFSM6AAAAAARPWOIHU>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

Lameck1 commented on index.html:
```diff
@@ -0,0 +1,36 @@
+      </div>
+    </nav>
+  </header>
+  <section class="heading">
+    <h1 id="name">
+      Kashif Saleem
+    </h1>
+  </section>
+  <section class="intro">
+    <p id="intro-heading">
+      Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+      If you like what you see and have a project you need coded, don’t hesitate and contact me.
+    </p>
+  </section>
+  <div class="button">
+    <a id="btn" href="">Start collaboraton</a>
+  </div>
```
You are almost there @Kashif-Saleem-Ghuman. Let's make the whole of this to be one section. Your name, the paragraph, and the button should all be part of one section. We often call this first section the `hero section`. It is the landing page of your portfolio.

Please refactor your code as below and we'll be good to go:

```JavaScript
<section class="heading">
  <h1 id="name">Kashif Saleem</h1>
  <p id="intro-heading">
    Hello! I am a software developer! I can help you build a product, feature, or website. Taka a look at my work.
    Don't hesitate to contact me if you like what you see and have a project you need to be coded.
  </p>
  
  <div class="button">
    <a id="btn" href="">Start collaboraton</a>
  </div>
</section>
```

Lameck1 reviewed (commented):
Hello @Kashif-Saleem-Ghuman,

This is a manual review.

Based on the very last review you received from @leonardodiasb, for you to address the concerns from the review, you need to consider the code example discussed in the inline comment below. Please do this last change and we'll go ahead and approve the project once you are done.

Cheers ✌🏼and happy coding.

Kashif-Saleem-Ghuman committed:
Name, the paragraph, and the button made  part of one section in  index.html and style.css updated accordingly

Abenezer-Tilahun reviewed (approved):
Hi @Kashif-Saleem-Ghuman  
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Abenezer-Tilahun commented:
Hello @Kashif-Saleem-Ghuman  I am really sorry for taking your time for misguided  you for pointing out use of `h1` tag for incorrect place.

Kashif-Saleem-Ghuman commented:
Abenezer I appreciate your apology. Apologies accepted. It did get me into a bit of panic but all is well that ends well.
Thank you.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

From: ***@***.***>
Sent: Thursday, October 27, 2022 7:22 PM
To: ***@***.***>
Cc: ***@***.***>; ***@***.***>
Subject: Re: [Kashif-Saleem-Ghuman/My-Portfolio-] My portfolio project pull request (PR #1)


Hello @Kashif-Saleem-Ghuman<https://github.com/Kashif-Saleem-Ghuman> I am really sorry for taking your time for misguided you for pointing out use of h1 tag for incorrect place.

—
Reply to this email directly, view it on GitHub<https://github.com/Kashif-Saleem-Ghuman/My-Portfolio-/pull/1#issuecomment-1293601611>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A2UBBJN4HKKZCO77WEGVRCDWFKF2VANCNFSM6AAAAAARPWOIHU>.
You are receiving this because you were mentioned.Message ID: ***@***.***>
